### ISSUE TITLE
Medium as Supported Account

### ISSUE THREAD

**Issue by AlexanderMann** [mudassir0909/jsonresume-theme-elegant#72]
Presently Medium.com isn't supported as one of the profiles. Not sure where this support should be added, but it'd be nice to have!

If you can list the files which need to be modified or the best path forward, I'd be happy to help!

**Comment by mudassir0909** [comment]
Well, the first step is to update [this file](https://github.com/jsonresume/theme-utils/blob/master/basics.js) to support Medium. Once that is done, the remaining tasks are pretty trivial

**Comment by mudassir0909** [comment]
Medium is now supported :)

**Comment by AlexanderMann** [comment]
Woah, awesome! Thanks @mudassir0909
